docs/ui/components/Search/styles.ts

```diff
@@ -0,0 +1,862 @@
+    --docsearch-searchbox-focus-background: ${theme.background.default};
+    --docsearch-searchbox-shadow: none;
+
+    /* hit */
+    --docsearch-hit-height: 56px;
+    --docsearch-hit-color: ${theme.text.secondary};
+    --docsearch-hit-active-color: ${theme.background.tertiary};
+    --docsearch-hit-background: ${theme.background.default};
+    --docsearch-hit-shadow: none;
+
+    /* footer */
+    --docsearch-footer-height: 44px;
+    --docsearch-footer-background: ${theme.background.secondary};
+  }
+
+  /* Dark Theme overwrites */
+
+  html[data-expo-theme='dark'] {
+    .DocSearch-Hits mark {
+      background: ${theme.palette.primary[200]};
+    }
+
+    .DocSearch-Hit[aria-selected='true'] mark {
+      background: ${theme.palette.primary[300]};
+    }
+
+    .DocSearch-Hit-icon {
+      opacity: 0.6;
+    }
+
+    .DocSearch-Hit-action-button:hover path,
+    .DocSearch-Hit-action-button:focus path {
+      fill: ${theme.palette.primary[600]};
+      stroke: ${theme.palette.primary[600]};
+    }
+
+    .DocSearch-Hit[aria-selected='true'] .DocSearch-Hit-icon:before {
+      filter: brightness(1.25);
+    }
+  }
+
+  /* === Button === */
+
+  .DocSearch-Button {
+    align-items: center;
+    background: var(--docsearch-searchbox-background);
+    border: 0;
+    border-radius: ${borderRadius.small}px;
+    color: var(--docsearch-muted-color);
+    cursor: pointer;
+    display: flex;
+    font-weight: 500;
+    height: 40px;
+    justify-content: space-between;
+    margin: 0 ${spacing[3]}px 0;
+    padding: ${spacing[0.5]}px ${spacing[3]}px;
+    user-select: none;
+    width: 24vw;
+    max-width: 320px;
+    box-shadow: ${shadows.input};
+    border: 1px solid ${theme.border.default};
+  }
+
+  .DocSearch-Button:hover,
+  .DocSearch-Button:active,
+  .DocSearch-Button:focus {
+    color: var(--docsearch-text-color);
+    outline: none;
+  }
+
+  .DocSearch-Button-Container {
+    align-items: center;
+    display: flex;
+  }
+
+  .DocSearch-Search-Icon {
+    stroke-width: 1.6;
+  }
+
+  .DocSearch-Button .DocSearch-Search-Icon {
+    color: ${theme.icon.default};
+    width: ${iconSize.small}px;
+    margin-right: ${spacing[1]}px;
+  }
+
+  .DocSearch-Button-Placeholder {
+    font-size: 1rem;
+    padding: 0 12px 0 6px;
+    color: ${theme.text.secondary};
+    opacity: 0.66;
+  }
+
+  .DocSearch-Button-Keys {
+    display: flex;
+    min-width: calc(2 * 20px + 2 * 0.4em);
+  }
+
+  .DocSearch-Button-Key {
+    display: flex;
+    align-items: center;
+    height: 20px;
+    ${typography.utility.pre}
+    ${kbdStyle}
+    margin-left: 0.4em;
+  }
+
+  @media (max-width: 768px) {
+    .DocSearch-Button {
+      width: 42px;
+    }
+
+    .DocSearch-Button-Keys,
+    .DocSearch-Button-Placeholder {
+      display: none;
+    }
+  }
+
+  /* === Modal === */
+  /* Body modifier */
+
+  .DocSearch--active {
+    overflow: hidden !important;
+  }
+
+  .DocSearch--active #__next {
+    filter: blur(6px);
```

Comment:
```suggestion
    filter: blur(2px);
```
I think we can dial this down. It's a nice effect but 6px is too much imo

Reply:
With, `2px` I was able to read most of the headers, which was a bit distracting, so I went with `3px`, let me know if this is fine. 😉

Reply:
That's okay! I am good with that